packages/wonder-blocks-modal/components/standard-modal.js

```diff
@@ -114,6 +115,15 @@ const styles = StyleSheet.create({
         background: Color.white,
         borderRadius: 4,
         overflow: "hidden",
+
+        // On mobile, we consume the full screen size.
+        [smOrSmaller]: {
+            width: "100%",
+            height: "100%",
+            maxWidth: "none",
```

Comment:
Ooh, I bet we also need to clear out `minHeight` here, or else the modal will overflow on not-tall-enough screens!

Reply:
Did this end up being necessary or not?

Reply:
Whoops, I missed this, thank you! Fixed in #72.